src/cg_ftoc.c

```diff
@@ -2949,19 +2921,9 @@ CGNSDLL void FMNAME(cg_gorel_f, CG_GOREL_F)(cgint_f *fn, cgint_f *ier, ...)
 
      /* read arguments */
     for (n = 0; n < CG_MAX_GOTO_DEPTH; n++)  {
-#ifdef _CRAY
```

Comment:
Why remove CRAY and WINDOWS parts

Reply:
As I understand it these parts are not needed anymore when *not* calling C varargs from Fortran code.